kahuna/public/js/crop/view.html

```diff
@@ -8,22 +8,22 @@
     <gr-top-bar-actions>
         <div class="top-bar-item side-padded">{{ ctrl.cropWidth() }} &times; {{ ctrl.cropHeight() }}</div>
 
-        <label class="top-bar-item clickable side-padded" title="landscape">
+        <label class="top-bar-item clickable side-padded titip-default titip-bottom" data-title="landscape [l]">
```

Comment:
Is that `landscape` -> `Landscape` or `[l]` -> `[L]`?

The latter implies needing to press `shift` to use the shortcut, which isn't correct.